feat(changelog): functions-added-serverless-python-api-framework 2023-05-12 (#1647)
scaleway/docs-content: changelog/may2023/2023-05-12-functions-added-serverless-python-api-framework.mdx

---
title: Serverless Python API Framework now supports offline testing
status: added
author:
    fullname: 'Join the #serverless-functions channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2023-05-12
category: serverless
product: functions
---

> Our Python-API-Framework, that enable you to simply create and deploy Python Serverless Functions using only Python code, now supports local testing with the command `scw-serverless dev app.py`.

Reviewer: ```suggestion
The Python-API-Framework enables you to create and deploy Python Serverless Functions using only Python code. It now supports local testing with the command `scw-serverless dev app.py`.
```

> For more information and examples check [the project docs](https://serverless-api-project.readthedocs.io/en/latest/)

Reviewer: ```suggestion
For more information and examples check [the project's documentation](https://serverless-api-project.readthedocs.io/en/latest/)
```
